modules/ocf/facts.d/puppetdb-running

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -uo pipefail
```

Comment:
We can also use `-e` if you're willing to squeeze the curl line onto the if, like

    if curl blah blah blah \
            blah blah blah \
            blah blah blah; then
        echo 'puppetdb running=true'

Just saying...